include/swift/Reflection/ReflectionContext.h

```diff
@@ -52,6 +52,25 @@ template <> struct MachOTraits<8> {
   using Section = const struct llvm::MachO::section_64;
   static constexpr size_t MagicNumber = llvm::MachO::MH_MAGIC_64;
 };
+
+template <unsigned PointerSize> struct ELFTraits;
+
+template <> struct ELFTraits<4> {
```

Comment:
Instead of the `4` can we specialize using `llvm::ELF::ELFCLASS32`?

Reply:
That was my implementation initially, but I saw that `MachOTraits` uses the `PointerSize` instead, so I decided to follow the same pattern for symmetry.

Tell me if you want me to change it, even if it will break the symmetry.

Reply:
I think that it is nicer to break symmetry as that is a pattern used in the LLVM side for ELF quite a bit.